src/ILCompiler.WebAssembly/src/CodeGen/WebAssemblyObjectWriter.cs

```diff
@@ -190,6 +195,25 @@ public void FinishObjWriter()
             //throw new NotImplementedException(); // This function isn't complete
         }
 
+        private void EmitDebugMetadata()
+        {
+            var dwarfVersion = LLVM.MDNode(new[]
+                        {
```

Comment:
Nit: Indentation seems to be off. (Also, a bunch of `if(` without a space, but I would get tired pointing them out.)

Reply:
Fixed